client/reader/sidebar/index.jsx

```diff
@@ -198,8 +197,7 @@ export const ReaderSidebar = createReactClass( {
 								</li>
 							) }
 							<ReaderSidebarTeams teams={ this.props.teams } path={ this.props.path } />
-							{ config.isEnabled( 'reader/conversations' ) &&
-							isAutomatticTeamMember( this.props.teams ) && (
+							{ isAutomatticTeamMember( this.props.teams ) && (
```

Comment:
lets double check this default to false when a user doesn't have a cache

Reply:
`export const isAutomatticTeamMember = teams => !! find( teams, [ 'slug', 'a8c' ] );`